week1/index.js

```diff
@@ -1,38 +1,31 @@
-import HTTP from 'http'
-import Path from 'path'
 
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
+"use strict"
 
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
 
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
-	
-	response.end()
-})
+let state = 10;
+let http = require("http");
+let server = http.createServer();
 
-server.listen(3001)
+server.on("connection", () => {
+    console.log("server is connected")
+})
 
-console.log('Server started')
\ No newline at end of file
+server.on("request", (request, response) => {
+    console.log("requesting", request.url)
+    function setResponse(header) {
+        response.setHeader("Content-Type", "Text-html");
+        response.write(header);
+        response.end();
+    }
+    switch (request.url) {
+        case "/add" : state++; setResponse(`<h1>${state}</h1>`); break;
```

Comment:
While I am usually a fan of 'terse' code, this is verging on difficult to read.
In-line case statements are useful when the case does only one thing, but in your case, you have two statements in here (`case "/add": state++; setResponse...`).

However, you might want to explore how to use expressions inside string templates, like:

    setResponse(`<h1>${ state++ }</h1>`);